src/op_mode/dhcp.py

```diff
@@ -128,12 +132,12 @@ def _get_raw_server_leases(family='inet', pool=None, sorted=None, state=[], orig
         data_lease['remaining'] = '-'
 
         if lease['valid-lft'] > 0:
-            data_lease['remaining'] = lease['expire_timestamp'] - datetime.utcnow()
+            data_lease['remaining'] = lease['expire_timestamp'] - datetime.now(timezone.utc)
```

Comment:
Does this not just produce the same output?

Reply:
datetime.utcnow() is deprecated since 3.12
https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow